tests/integration/sshd/sshd_spec.rb

```diff
@@ -28,4 +28,15 @@
     it { should be_listening }
     its('processes') { should include(/sshd/) }
   end
+
+  describe file('/etc/ssh/ssh_host_ed25519_key') do
+    its('mode') { should cmp '0600' }
+    its('content') { should match /^\s*PRIVATE KEY CONTENT$/ }
```

Comment:
Mmh, the `\s*` might be counter productive, as it actually should show up without whitespaces in the file. Was this included on purpose? Otherwise, `^PRIVATE...` is better I think.

Reply:
I just copy/pasted from other test cases, ... the `\s` should probably go away as you indicated. Ideally I would like to validate the exact content of the whole file. Is there a suitable regexp for that or would I go with some other matching construct?